test/compat-tests.jl

```diff
@@ -42,7 +42,6 @@ bbv = Blobs.malloc_and_init(BlobBitVector, 3)
 pbv = @v bbv
 pbv[2] = true
 @test pbv[2] == true
-@test pv[2] == Foo(2, 2.2)
```

Comment:
Dup of the test above IIUC?

Reply:
yeah exactly. i think this was a copy/paste error and doesn't belong here.